src/main/java/use_case/login/LoginInputData.java

```diff
@@ -0,0 +1,47 @@
+package use_case.login;
+
+import use_case.logout.LogoutInputBoundary;
+
+public class LoginInputData {
+
+    private String userID;
+    private String userName;
+    private String password;
+    private String email;
+    private String birthData;
+    private String fullName;
+
+    public LoginInputData(String userID, String userName, String password,
```

Comment:
Yeah, that's right the user only need username and password to login. I included userID, birthdate and full name, because based on the UML diagram of the login use case, they need to be included. So, I’m not entirely sure if there are specific reasons for their inclusion. I think I will change it to just username and password for now and also to avoid the merge conflicts in the entity folder.

Reply:
Hmm I think we should login through email, since usernames could overlap? That's how we implemented sign up, we can't sign up if the entered email exists

Reply:
Oh, I thought we were going to make usernames unique. I will change it to email instead.